packages/playground/website/src/components/import-button/index.tsx

```diff
@@ -8,6 +8,7 @@ export default function ImportButton() {
 	const [isOpen, setOpen] = useState(false);
 	const openModal = () => setOpen(true);
 	const closeModal = () => setOpen(false);
+	const importButtonAriaLabel = 'Open Playground import window';
```

Comment:
This text is redundant.

```suggestion
	const importButtonAriaLabel = 'Import configuration';
```

Reply:
+1